cht-config/package.json

```
@@ -12,18 +12,20 @@
     "test-tasks": "npm run eslint && TZ=Africa/Nairobi mocha --reporter progress test/tasks/*.spec.js --timeout 10000",
     "test-targets": "npm run eslint && TZ=Africa/Nairobi mocha --reporter progress test/targets/*.spec.js --timeout 10000",
     "test-contact-summary": "npm run eslint && TZ=Africa/Nairobi mocha --reporter progress test/contact-summary/*.spec.js --timeout 10000",
-    "test-unit": "TZ=Africa/Nairobi mocha --recursive --reporter spec test --timeout 20000"
+    "test-unit": "TZ=Africa/Nairobi mocha --recursive --reporter spec test --timeout 20000",
+    "deploy": "wait-on http://api:5988/ && && sleep 100 && cht --url=http://admin:password@api:5988 --force"
```

Docker failed locally with `sh: syntax error: unexpected "&&"` because of the double `&&`
```suggestion
    "deploy": "wait-on http://api:5988/ && sleep 100 && cht --url=http://admin:password@api:5988 --force"
```